content/en/agent/docker/apm.md

```diff
@@ -44,7 +44,7 @@ List of all environment variables available for tracing within the Docker Agent:
 | ------                     | ------                                                                                                                    |
 | `DD_API_KEY`               | [Datadog API Key][1]                                                                                                      |
 | `DD_PROXY_HTTPS`           | Set up the URL for the proxy to use.                                                                                      |
-| `DD_HOSTNAME`              | Set the Agent hostname manually.                                                                                          |
+| `DD_HOSTNAME`              | Manually set the hostname to use for metrics if autodection fails, or running the DCA.                                    |
```

Comment:
The table formatting will change so I'm not using GitHub suggestions but I suggest:

`Manually set the hostname to use for metrics if autodection fails, or when running the Datadog Cluster Agent.`